### PR TITLE
Cso dash update

### PR DESCRIPTION
**Pull request by drzackyll**

### CODE REVIEW

**Comment by drzackyll** [comment]
connects unicef/etools-issues#966

File: dash/app-elements/dashboard/dashboard-partnerships.html [hunk 1]
```diff
@@ -42,39 +49,105 @@
...
       }
 
-       data-table-row span:nth-child(3) {
-         white-space:normal;
-         overflow-y: auto;
-      }
       div.sections{
         height: 48px;
-        line-height: 48px;
+        white-space: normal !important;
```
Comment (marko911) [inline dash/app-elements/dashboard/dashboard-partnerships.html]: avoid !important selectors

File: dash/app-elements/dashboard/dashboard-partnerships.html [hunk 2]
```diff
@@ -660,12 +781,19 @@
           let nonZero = this.displayNonZero(val);
           return Number(nonZero) ? this.currencyFormat(nonZero) : nonZero;
         },
-        _isEqual: function(total, disbursed) {
-          // return disbursed === total
-          return true;
+        _isEqual: function(first, second) {
```
Comment (marko911) [inline dash/app-elements/dashboard/dashboard-partnerships.html]: remove this, lodash has a _.isEqual function

File: dash/app-elements/dashboard/dashboard-partnerships.html [hunk 3]
```diff
@@ -660,12 +781,19 @@
...
+          return first === second;
+          // return true;
+        },
+        _checkAllCurrenciesConsistent: function(all_currencies_are_consistent, fr_currencies_are_consistent) {
+          if (!all_currencies_are_consistent) {
```
Comment (marko911) [inline dash/app-elements/dashboard/dashboard-partnerships.html]: return a tertiery here

**Pull request merged**